setup.py

```diff
@@ -56,6 +56,7 @@
     ],
     install_requires=[
         'h11 ~= 0.8.1',  # means: 0.8.x where x >= 1
+        'h2 >= 3.1.0',
```

Comment:
Should we have an upper bound here?

Reply:
We could do `'h2 ~= 3.1'`?

Reply:
I never really know/remember what `~=` means.
I think h2 is a proper good semver citizen and we can use `>=3.1,<4`.

Reply:
`>=3.1,<4` is the same as `~=3.1` as I understand it.

Reply:
👍